src/App.vue

```diff
@@ -643,6 +656,11 @@ export default class App extends Mixins(
     // NB: will throw if API error
     let draftFiling = await LegalServices.fetchDraftDissolution(this.getBusinessId)
 
+    // SB TODO remove once filing is available
+    draftFiling.business = { ...draftFiling.business,
```

Comment:
overriding filing type as SP for testing purposes. will remove once create is in dev

Reply:
Obsolete review comment? If so please delete it.